packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -33,39 +33,97 @@ import {
 export async function runBrokenLinks(
   options: BrokenLinksResultV1_BrokenLinkCheckerOptions
```

Comment:
I thought that we had discussed having a different interface for the runBrokenLinks options that were separate from those that were actually ran (ie defaults are set to). What's the plan with that?

Reply:
that PR just got submitted: https://github.com/GoogleCloudPlatform/synthetics-sdk-nodejs/pull/26